File: C++/fizz-buzz.cpp
#include<iostream>
#include<string>
using namespace std;

int main(){
	for(int i = 1;i <= 100;i++){
		string output = "";
		output += i % 3 == 0 ? "Fizz" : "";
		output += i % 5 == 0 ? "Buzz" : "";
		if(output == "") output = to_string(i);
		cout << output << endl;
	}
	return 0;
}

File: C#/fizz-buzz.cs
static public void Main()
    {
        for (int i =1; i < 100; i++)
        {
            Console.WriteLine((i%3==0 && i%5==0)?"Fizz Buzz":i%3==0 ?"Fizz": i%5==0 ?"Buzz":i.ToString());
        }

    }
